infra/general/languageCodeManager.php

```diff
@@ -82,6 +82,14 @@ public static function getObjectFromKalturaName($kalturaName)
         return self::getObjectFromTwoCode($val);
     }
 
+	public static function getThreeCodeFromKalturaName($kalturaName)
+    {
+        if(!self::isAlreadyLoaded())
```

Comment:
since we are already doing this in getObjectFromKalturaName this if condition is not really needed